watertap/data/techno_economic/zeroth_order_model_data/anaerobic_digestion_oxidation.yaml

```diff
@@ -0,0 +1,20 @@
+default:
+  default_removal_mass_solute:
+    unit: kg/kg
+    value: 0
+  energy_electric_flow_vol_inlet:
+    unit: kWh/m^3
+    value: 0.15
+  recovery_vol:
+    reference: 'Beef Processing case study. '
+    unit: m^3/m^3
+    value: 0.9999
+  removal_frac_mass_solute:
+    bod:
+      constituent_longform: Biological Oxygen Demand (BOD)
+      unit: kg/kg
+      value: 0.0009699999999999999
```

Comment:
Minor comment - is this just 0.00097 represented as a float (due to binary precision)? It probably doesn't matter in the end, but it would be nice to use the actual value if possible (rather than the nearest float).

Reply:
Hmm not sure. These were auto imported and converted. I did not put precision limits on these imports, I will look into adding that into next commit.

Reply:
Skimming through it looks like the longest number has 9 significant figures (some values of `energy_electric_flow_vol_inlet`).